docs/eigenda/operator-guides/requirements/delegation-requirements.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 2
+---
+
+# Delegation Requirements
+
+:::info 
+Active Operator Set Cap: the active operator set cap is currently
+limited to 200 operators. We will review this operator cap on an ongoing basis,
+and explicitly communicate future changes.
+:::
+
+EigenDA currently only allows a limited number of operators to join the protocol. This restriction relates directly to the cost of bridging EigenDA availability attestations to the Ethereum L1 and will be relaxed as this cost is alleviated via various technological improvements. 
+
+The EigenDA operator cap means that in order to be an operator for EigenDA, you must first register as an EigenLayer operator and meet certain stake requirements. These requirements are evaluated on a per-quorum basis, relative to the weighting for each quorum: 
+- Minimum stake floor: Each operator must have a threshold quantity of stake. 
+- Congested operator set stake floor: When the global operator cap (N) is reached, operators must be within the top N operators by quorum weight in order to join the quorum. In order to limit excessive churn of the operator set, certain additional requirements are also enforced, as described in the next section.
+
+## EigenDA Churn Management Information
+
+### Churn Approver
```

Comment:
The content below has more details on how it works than what is required. Should we keep it short? At least, this doesn't need to be under Delegation Requirements (as it's like some appendix/expanded content for internals)

Reply:
I agree that this section in in need of updates. I'd like to address this in a separate PR so that we are not blocking the refactoring work that is the main objective of this PR.